.rustfmt.toml

```diff
@@ -9,7 +9,6 @@ format_code_in_doc_comments = false
 comment_width = 80
 normalize_comments = true # changed
 normalize_doc_attributes = false
-license_template_path = "FILE_TEMPLATE" # changed
```

Comment:
I never really noticed but does removing this mean no checking that the license header is at the top of files now?

Reply:
No it has been removed now so it doesn't do anything other than give a warning https://github.com/rust-lang/rustfmt/issues/5103